01-intro.Rmd

```diff
@@ -234,7 +234,7 @@ Data visualization domain is vast and an aspiring visualizes, a person who can l
 * Attend events, seminars and join conferences conducted locally/ globally
 * Give back to the community once you become an expert in that field
 
-There are very active Tableau social media groups [@LinkedIn_Groups]:
+There are very active Tableau social media groups [@inkedIn_Groups]:
```

Comment:
@pragyansharma24  I see that you've added a new reference to "@inkedIn_Groups" but in the code added to conclusion.rmd, the reference is "@LinkedIn_Groups". Correct me if I'm wrong, but shouldn't both tags be "@LinkedIn_Groups"

Reply:
Thank you Jason @82817 . That was typing mistake. I will fix this one.